### PR TITLE
ci: Various updates

### PR DESCRIPTION
**Pull request by akash1810**
> **Note**
> Easiest to review commit by commit.

## What does this change?
A few updates to CI:
  - Build using Java Corretto, as that is what we use at runtime
  - Apply prettier across the CDK files, after noticing inconsistent quote marks
  - Updates GuCDK to the latest version
  - Removes private information from VCS

## How to test
  - The build should pass
  - The application should be deployable
  - Once deployed, logs from the healthcheck should be visible in Central ELK

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -38,19 +38,18 @@ jobs:
...
+
       - name: Build
         run: |
           ./scripts/build.sh
-          
+
       - name: Upload to Riff-Raff
         uses: guardian/actions-riff-raff@v2
         with:
-          app: cv-redact-tool
```
Comment (akash1810) [inline .github/workflows/build.yml:52]: `app` is redundant as it is ignored when `projectName` is set - https://github.com/guardian/actions-riff-raff#available-inputs.

File: cdk/jest.setup.js [hunk 2]
```diff
@@ -1 +1,2 @@
-jest.mock("@guardian/cdk/lib/constants/tracking-tag");
+jest.mock('@guardian/cdk/lib/constants/tracking-tag');
+jest.mock('@guardian/private-infrastructure-config');
```
Comment (akash1810) [inline cdk/jest.setup.js:2]: Adds the Jest mock - https://github.com/guardian/private-infrastructure-config#using-with-jest.

File: cdk/lib/__snapshots__/cv-redact-tool.test.ts.snap [hunk 3]
```diff
@@ -712,42 +714,6 @@ dpkg -i /cv-redact-tool/cv-redact-tool.deb",
...
-        },
-        "PolicyName": "ssm-run-command-policy",
-        "Roles": [
-          {
-            "Ref": "InstanceRoleCvredacttool834C6B7C",
-          },
-        ],
-      },
-      "Type": "AWS::IAM::Policy",
-    },
```
Comment (akash1810) [inline cdk/lib/__snapshots__/cv-redact-tool.test.ts.snap:750]: GuCDK v49.0.2 moves to using the AWS managed policy for SSM access - https://github.com/guardian/cdk/releases/tag/v49.0.2

**Review by mchv — APPROVED** [review]

File: cdk/jest.setup.js (hunk 2 above)
Comment (mchv) [reply cdk/jest.setup.js:2]: Thanks, I add missed this!

**Pull request merged**